examples/Makefile

```diff
@@ -489,7 +489,8 @@ SHADERS = \
     shaders/shaders_spotlight \
     shaders/shaders_hot_reloading \
     shaders/shaders_mesh_instancing \
-    shaders/shaders_multi_sample2d
+    shaders/shaders_multi_sample2d \
+    shaders/shaders_compute_gol
```

Comment:
I prefer to avoid it in the Makefile, this examples is OpenGL 4.3 specific so compilation will fail in some platforms.